src/item.rs

```diff
@@ -934,6 +935,8 @@ impl Item {
             #[cfg(feature = "vic3")]
             Item::DynamicCountryName => "common/dynamic_country_names/",
             #[cfg(feature = "vic3")]
+            Item::EventCategory => "events/",
```

Comment:
Since it's defined in an internal table  the path here should be "". The path should be events iff the item values are collected from data in events/